functions.php

```diff
@@ -152,11 +151,43 @@ function twentynineteen_scripts() {
  * Enqueue supplemental block editor styles
  */
 function twentynineteen_editor_frame_styles() {
+
+	// Include color patterns
+	require_once( get_parent_theme_file_path( '/inc/color-patterns.php' ) );
+
 	wp_enqueue_style( 'twentynineteen-editor-frame-styles', get_theme_file_uri( '/style-editor-frame.css' ), false, '1.0', 'all' );
+	wp_add_inline_style( 'twentynineteen-editor-frame-styles', twentynineteen_custom_colors_css() );
```

Comment:
Ahh, good catch! Agreed that there’s probably a more efficient way to get Gutenberg to catch these theme color overrides.

Reply:
The best way I've found for passing dynamic CSS to the new editor is to add it to the `styles` array in https://github.com/WordPress/gutenberg/blob/master/lib/client-assets.php#L1691

Registered editor stylesheets get processed here: https://github.com/WordPress/gutenberg/blob/master/lib/client-assets.php#L1567-L1581

The same thing can be done by passing the generated CSS string directly instead of a file. This will ensure that the CSS gets wrapped/scoped when #262 is merged.

Reply:
I’m not sure I’m following here @spencerfinnell  

> The best way I've found for passing dynamic CSS to the new editor is to add it to the styles array in https://github.com/WordPress/gutenberg/blob/master/lib/client-assets.php#L1691

> The same thing can be done by passing the generated CSS string directly instead of a file. 

When I pass an array with the css string to `add_editor_style()`, I get an “expects parameter 1 to be string” error. Can you explain in a little more detail or offer a snippet on how you did these two things?